### PR TITLE
Fix Helm chart naming

### PR DESCRIPTION
**Pull request by zeenix**
Please see the discussion here for rationale of these changes: https://github.com/habitat-sh/habitat/pull/4638

### CODE REVIEW

**Comment by asymmetric** [comment]
A [commit message](https://github.com/kinvolk/habitat/pull/23/commits/78fdb9972fa5813acea72c5727c8d517d9686460) says

> We'll need it to create `--help` string for `--version` CLI option in a
following patch.

Is this supposed to happen in this PR?

File: components/pkg-export-helm/Cargo.toml [hunk 1]
```diff
@@ -19,6 +19,7 @@ habitat_pkg_export_docker = { path = "../pkg-export-docker" }
 habitat_pkg_export_kubernetes = { path = "../pkg-export-kubernetes" }
 # We need to lock here since v0.30.0 bumps to a version of pest that fails to build on Windows.
 handlebars = { version = "0.29.1", default-features = false }
+lazy_static = "*"
```
Comment (asymmetric) [inline components/pkg-export-helm/Cargo.toml:22]: I'd rather pin to a specific version, or at least use a less relaxed version constraint. The [README](https://github.com/rust-lang-nursery/lazy-static.rs#getting-started) suggests `=1.0`, and who am I to disagree?

**Comment by zeenix** [comment]
>     Is this supposed to happen in this PR?

It is, just that github messed up the order of commits. :(

File: components/pkg-export-helm/Cargo.toml (hunk 1 above)
Comment (zeenix) [reply components/pkg-export-helm/Cargo.toml:22]: I just copy&pasted the line from other components and I'd like to be consistent with them.

**Comment by zeenix** [comment]
Closing for https://github.com/habitat-sh/habitat/pull/4681

**Pull request closed without merging**